pkg/commands/policy/policy.go

```diff
@@ -90,5 +96,40 @@ func (c *PolicyCommand) Run(ctx context.Context, args []string) error {
 		return fmt.Errorf("failed to unmarshal json: %w", err)
 	}
 
-	return nil
+	c.results = results
+
+	return c.Process(ctx)
+}
+
+// Process handles the main logic for handling the results of the policy
+// evaluation.
+func (c *PolicyCommand) Process(ctx context.Context) error {
```

Comment:
You could accept the results as a param here or move the code process the results file into the process function and use a file from a testdata folder inside this dir to test the functionality fully.

Reply:
Opted for latter